templates/base/babel.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   presets: ['@babel/preset-env', '@babel/preset-react'],
   plugins: [
```

Comment:
We need to setup environment specific babel configs :)

Reply:
@ankurkakroo2 I believe that will come from our NPM module. We can fix it here too for time being

Reply:
done, moved transform-flow-strip-types to test env